consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/IbftRoundFactory.java

```diff
@@ -24,22 +24,22 @@
 import tech.pegasys.pantheon.util.Subscribers;
 
 public class IbftRoundFactory {
-
   private final IbftFinalState finalState;
-
   private final IbftBlockCreatorFactory blockCreatorFactory;
   private final ProtocolContext<IbftContext> protocolContext;
   private final ProtocolSchedule<IbftContext> protocolSchedule;
-  private final Subscribers<MinedBlockObserver> minedBlockObservers = new Subscribers<>();
+  private final Subscribers<MinedBlockObserver> minedBlockObservers;
```

Comment:
With `Subscribers` being a mutable object passed  by reference to the constructor, you are allowing that object to be altered (the subscriptions to be added / removed) after the time of the factory construction. Is this the intention?

Reply:
It definitely wasn't the intention, it doesn't need to be mutable. The IbftRound just needs to iterate the list of subscribers.